.travis/travis-functions.sh

```diff
@@ -48,3 +48,26 @@ function createRelease() {
   # releasing graph-services
   sbt "release skip-tests default-tag-exists-answer k with-defaults"
 }
+
+function updateVersionInRenku() {
+  echo "Updating renku-graph version in Renku's requirements.yaml"
+
+  # cloning the renku repo
+  cd || exit
+  git clone https://github.com/SwissDataScienceCenter/renku.git
+  cd renku || exit
+
+  # changing the version
+  sed -i "/- name: renku-graph/{n;n;n;s/.*/  version: $TRAVIS_TAG/}" charts/renku/requirements.yaml
+
+  # preparing git config
+  git config --global user.name "RenkuBot"
+  git config --global user.email "jakub.chrobasik@epfl.ch"
```

Comment:
shouldn't a more generic/sdsc email be used here?

Reply:
I agree -- use renku@datascience.ch

Reply:
Cool! I didn't even know we have that.